c/doc/run_sample_on_toradex_wec2013.md

```diff
@@ -0,0 +1,75 @@
+# Getting started - running a C sample application on Windows Embedded Compact 2013 on a Toradex module
```

Comment:
I think referencing Toradex here makes absolute sense, as this is a specific platform "run on" doc.